src/request/mod.rs

```diff
@@ -43,7 +42,7 @@ pub struct Request<'a, 'b: 'a> {
     pub headers: Headers,
 
     /// The request body as a reader.
-    pub body: Body<'a, 'b>,
+    pub body: Cursor<Vec<u8>>,
```

Comment:
This seems a bit scary - we'll want to be able to handle streaming request bodies.